user-onboarding/cypress/e2e/form_test.cy.js

```diff
@@ -0,0 +1,273 @@
+    // .should('be.disabled')
+
+    //Tried this as well and came up with the same error:
+    // termsCheckbox()
+    // .should('have.prop', 'disabled', false)
+    // .click()
+    // .should('have.prop', 'disabled', true)
+  });
+
+  it('input data + submit form?', () => {
+    cy.contains('Zelda Trinity').should('not.exist')
+
+    firstNameInput()
+      .type('Zelda')
+      .should('have.value', 'Zelda')
+
+    lastNameInput()
+      .type('Trinity')
+      .should('have.value', 'Trinity')
+
+    emailInput()
+      .type('zelda@trinity.com')
+      .should('have.value', 'zelda@trinity.com')
+
+    passInput()
+      .type('Wizard$9')
+      .should('have.value', 'Wizard$9')
+
+    confirmPassInput()
+      .type('Wizard$9')
+      .should('have.value', 'Wizard$9')
+
+    termsCheckbox()
+      .click()
+
+    submitButton()
+      .click()
+
+    cy.contains('Zelda Trinity').should('exist')
+  });
+
+  it('Validation Errors after typing in input boxes', () => {
+    firstNameValErr()
+      .should('be.hidden')
+    firstNameInput()
+      .type('a')
+    firstNameValErr()
+      .should('not.be.hidden')
+      .should('have.text', 'First name must be at least 3 characters long')
+
+    lastNameValErr()
+      .should('be.hidden')
+    lastNameInput()
+      .type('b')
+    lastNameValErr()
+      .should('not.be.hidden')
+      .should('have.text', 'Last name must be at least 2 characters long')
+
+    emailValErr()
+      .should('be.hidden')
+    emailInput()
+      .type('c')
+    emailValErr()
+      .should('not.be.hidden')
+      .should('have.text', 'Enter a valid email')
+
+    pwValErr()
+      .should('be.hidden')
+    passInput()
+      .type('d')
+    pwValErr()
+      .should('not.be.hidden')
+      .should('have.text', 'Password must contain at least 8 characters, one uppercase, one number and one special case character')
+
+    confirmPwValErr()
+      .should('be.hidden')
+    confirmPassInput()
+      .type('e')
+    confirmPwValErr()
+      .should('not.be.hidden')
+      .should('have.text', 'Passwords do not match')
+  });
+})
+
+describe('Stretch - form on load', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+
+  it('Loads page correctly', () => {
+    cy.get('form').should('exist')
+
+    cy.get('div[class="user-container container"]')
+      .should('exist')
+
+    cy.get('div[class="user container"]')
+      .should('exist')
+
+    cy.get('div[class="first-name"]')
+      .should('contain', 'First Name')
+
+    cy.get('div[class="last-name"]')
+      .should('contain', 'Last Name')
+
+    cy.get('div[class="email"]')
+      .should('contain', 'Email')
+
+    cy.get('div[class="role"]')
+      .should('contain', 'Role')
+
+    cy.get('div[class="password"]')
+      .should('contain', 'Password')
+
+    //Can't confirm that Confirm Password is on the form because I made its class to be password (same as Password's class)
```

Comment:
You can either do 1 of two ways:
1) add a data-cy="confirmPassword" in the React code for the component 
2) cy.get('.password-confirm > :nth-child(2) > label').should('contain', 'Confirm Password')